recipe/meta.yaml

```diff
@@ -1,13 +1,12 @@
-{% set version = "1.9.5" %}
+{% set version = "1.10.0a2" %}
 
 package:
   name: holoviews
   version: {{ version }}
 
 source:
-  fn: holoviews-{{ version }}.tar.gz
-  url: https://github.com/ioam/holoviews/archive/v{{version}}.tar.gz
-  sha256: 89bdf9708ab58b43d4c690846867be5559a9cb4b7c215611db646ed3d54c4169
+  git_url: https://github.com/ioam/holoviews.git
+  git_rev: v{{version}}
```

Comment:
Could add `git_depth: 1` to make this a shallow clone. Think that should still work ok since we are picking the tag, but it would be worth checking first.

Reply:
Sure. I don't think a deep clone is needed either.

Reply:
Addressed in f3663c8